File: 프로그래머스/lv2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    answer = []
    zero_count, cal_count = 0, 0
    while len(s) > 1:
        zero_count += s.count('0')
        s = bin(s.count('1'))[2:]
        cal_count += 1

    answer.append(cal_count)
    answer.append(zero_count)
    return answer 

File: 프로그래머스/lv2/70129. 이진 변환 반복하기/이진 변환 반복하기.java
class Solution {
    public int[] solution(String s) {
    int[] answer = new int[2];
    int zeroCount = 0, calCount = 0, len = 0;
    while (s.length() > 1) {
        zeroCount += s.chars().filter(c -> c == '0').count();
        s = s.replace("0", "");
        s = Integer.toBinaryString(s.length());
        calCount++;
    }
    answer[0] = calCount;
    answer[1] = zeroCount;
    return answer;
    }
}